pkgs/development/python-modules/psrpcore/default.nix

```
@@ -42,5 +42,6 @@ buildPythonPackage rec {
     changelog = "https://github.com/jborean93/psrpcore/blob/v${version}/CHANGELOG.md";
     license = licenses.mit;
     maintainers = with maintainers; [ fab ];
+    platforms = platforms.linux;
```

```suggestion
    broken = stdenv.isDarwin;
```

with `stdenv` added to arguments. The package actually worked in April: https://hydra.nixos.org/job/nixpkgs/trunk/python312Packages.psrpcore.aarch64-darwin